app/views/devise/registrations/_edit_password.html.haml

```diff
@@ -1,23 +1,25 @@
-= form_for(resource, :as => resource_name, :url => registration_path(resource_name), :html => { :method => :put, :class => 'form-horizontal' }) do |f|
+= form_for(resource, as: resource_name,
+        url: registration_path(resource_name),
```

Comment:
Indentation here differs from the corresponding line in other files (eg `_edit_email.html.haml`): I don't know which one is right.